### PR TITLE
chore(release): reflect latest patched releases

### PR DESCRIPTION
**Pull request by jekyllbot**
After the releases of patched versions for 3.8, 3.7 and 3.6 branches, master should reflect those changes.

### CODE REVIEW

File: History.markdown [hunk 1]
```diff
@@ -110,14 +110,18 @@
...
   * Security: fix `include` bypass of `EntryFilter#filter` symlink check (#7226)
-  * 3.8.x: security: fix `include` bypass of `EntryFilter#filter` symlink check (#7228)
-  * 3.6.x: security: fix `include` bypass of `EntryFilter#filter` symlink check (#7229)
 
 ### feature
 
   * Disable Liquid via front matter (#6824)
   * Do not process Liquid in post excerpt when disabled in front matter (#7146)
 
+## 3.8.4 / 2018-06-18
```
Comment (ashmaroli) [inline History.markdown]: it should be `2018-09-18` here..

File: History.markdown [hunk 2]
```diff
@@ -401,6 +411,12 @@
   * Improve docs styling for code to be run in shell (#6641)
   * Fix permalink icon markup in news-item layout (#6639)
 
+## 3.6.3 / 2018-06-18
```
Comment (ashmaroli) [inline History.markdown]: it should be 2018-09-19 here..

File: History.markdown [hunk 3]
```diff
@@ -235,6 +239,12 @@
   * Handle liquid tags in excerpts robustly (#6891)
   * Allow front matter defaults to be applied properly to documents gathered under custom `collections_dir` (#6885)
 
+## 3.7.4 / 2018-09-07
+
+### Bug Fixes
+
+  * Security: fix `include` bypass of EntryFilter#filter symlink check
```
Comment (ashmaroli) [inline History.markdown]: link to PR..?

**Comment by DirtyF** [comment]
@jekyllbot: merge

**Pull request merged**